### PR TITLE
Only run rainbow highlighting test on 64-bit Unix

### PR DESCRIPTION
**Pull request by lnicola**

### CODE REVIEW

File: crates/ide/src/syntax_highlighting/tests.rs [hunk 1]
```diff
@@ -915,6 +915,10 @@ fn main() {
 }
 
 #[test]
+#[cfg_attr(
+    all(unix, not(target_pointer_width = "64")),
```
Comment (mati865) [inline crates/ide/src/syntax_highlighting/tests.rs:919]: Contrary to what title says this runs this test on UNIXes that are **not** 64-bit 😄

Comment (lnicola) [reply crates/ide/src/syntax_highlighting/tests.rs:919]: :scream_cat: no, it actually runs it on 32-bit non-Unix.

Comment (mati865) [reply crates/ide/src/syntax_highlighting/tests.rs:919]: Oh, sorry I thought it's normal `cfg`. Didn't see that `ignore` part 🤦🏻‍♂️

Comment (lnicola) [reply crates/ide/src/syntax_highlighting/tests.rs:919]: Well, it's still wrong 😀, thanks for spotting it.

**Pull request merged**